src/main/resources/mail/templates/shared/CAREIssueAlertTemplate.mustache

```diff
@@ -0,0 +1,81 @@
+<!doctype html>
+<html>
+<head>
+    <title>CARE Issue{{subtitle}}</title>
+    <style>
+
+      @import url("//fonts.googleapis.com/css?family=Inter:400,600");
+
+      h1 {
+        font-family: 'Inter', Arial, sans-serif;
+        font-weight: 600;
+        font-size: 18px;
+        line-height: normal;
+        color: #555e6f;
+        text-transform: uppercase;
+      }
+
+      .header {
+        font-size: 16px;
+        text-transform: uppercase;
+        line-height: normal;
+        color: #8e929b;
+        padding-top: 10px;
+      }
+
+      body {
+        font-family: 'Inter', Arial, sans-serif;
+        line-height: 1.5;
+        color: #333;
+      }
+
+      .divider {
+        width: 100%;
+        background-color: #e1e1eb;
+        height: 1px;
+        margin: 20px 0;
+      }
+
+      .detail {
+        font-family: 'Inter', Arial, sans-serif;
+        padding-top: 5px;
+      }
+
+      .footer {
+        padding-top: 10px;
+        border-top: 1px solid #e1e1eb;
+        margin-top: 30px;
+        color: #717175;
+      }
+
+    </style>
+</head>
+
+<body>
+    <div>
+        <h1>{{title}}</h1>
+        <div class="detail">
+            <a href="https://openlattice.com/care/#/issue/{{issueId}}">View issue</a>
```

Comment:
I don't think this will actually work because of mustache template URL escaping. CodexAlertEmailRenderer / CodexMessageAlertTemplate have an example of how to make this work -- generally speaking you'll want your mustache template to have something like
```
<div class="detail">
   {{{urlTag}}}
</div>
```
(with a triple curly bracket)
and then have your renderer assign the tag with a URL, eg
```
templateObjects["urlTag"] = "<a href=\"http://openlattice.com/blah\">View issue</a>"
```
```

Reply:
btw, `issueId` is a UUID, yeah? If it's a user input string then this is potentially unsafe, I think